### ISSUE TITLE
"Object reference not set to an instance of an object" when adding phone numbers

### ISSUE THREAD

**Issue by SuperDJ** [SuperOffice/SuperOffice.WebApi-Samples#12]
While using `POST` for creating a new contact/ company (https://sod.superoffice.com/customer-id/api/v1/Contact) I receive the following error:
> Object reference not set to an instance of an object

The body is below. It is formatted according the [docs](https://docs.superoffice.com/api/reference/restful/rest/Contact/v1ContactEntity_PostContactEntity.html#sample-request):
```json
{
  "Name": "New Company",
   "Department":null,
   "OrgNr":null,
   "Number2":10079,
   "Country": {
       "Name": "Netherlands"
  },
  "Address": {
     "Postal": {
          "Address1":"straat 1",
          "Zipcode":"1224 AD",
          "City":"Dracht"
       }
  },
  "Associate": {
       "AssociateId":22
   },
  "Phones": [
       {
        "Value":"1234567890",
        }
   ]
}
```

But when I remove the `Phones` part it does work:
```json
{
  "Name":"New Company",
   "Department":null,
   "OrgNr":null,
   "Number2":10079,
   "Country": {
       "Name":"Netherlands"
  },
  "Address": {
     "Postal": {
          "Address1":"straat 1",
          "Zipcode":"1224 AD",
          "City":"Dracht"
       }
  },
  "Associate": {
       "AssociateId":22
   }
}
```
How should a phone number be added to the body?

**Comment by xt1** [comment]
```json
   "Country": {
       "Name": "Netherlands"
   },
```
 
needs an Id value - then the Phone number can be prefixed correctly, and the save will succeed.

**Comment by SuperDJ** [comment]
@xt1  Adding `CountryId` to `Country` does seem to work yes.

**Comment by SuperOfficeDevNet** [comment]
This is has been fixed in 10.1.1, and will be released in online within 3 weeks.
